build.go

```diff
@@ -159,6 +159,11 @@ type BuildOptions struct {
 
 	// User's group id used to build the image
 	GroupID int
+
+	// Slice of two images
+	// 1. A previous image to set to a particular tag reference, digest reference, or image ID;
+	// 2. The image to refer to
+	Images []string
```

Comment:
I'm a little concerned about the direction that this PR is headed.. The previous image flag is intended to have space for one `string`, not two. If you see https://github.com/buildpacks/pack/blob/5ea76488f9c16bca671dd417da1f311f423a7d8b/build.go#L72, you'll see where that `<image-name>` was _already_ being collected as part of the pack build command.

Happy to clarify, if I wasn't clear!

Reply:
Forgive me, I'm having trouble understanding.

If you're asking how do I know `PreviousImage` was passed in by a user? You could assert that the value is not equal to an empty string (`""`). 

If you wanna talk about these things in a more realtime way, we're all pretty active at: https://slack.buildpacks.io/